gear-programs/vft-treasury/src/wasm/tests/end2end.rs

```diff
@@ -0,0 +1,89 @@
+use gclient::{GearApi, Result};
+mod utils_gclient;
+use sails_rs::{ActorId, H160, U256};
+use utils_gclient::*;
+
+#[tokio::test]
+#[ignore]
+async fn test_treasury() -> Result<()> {
+    let mut client = GearApi::vara_testnet().await?;
```

Comment:
```suggestion
    // It will not work on local node as vft-treasury logic relies on pallet-gear-eth-bridge
    // which will be initialized only in ~12 hrs from start on local node.
    let mut client = GearApi::vara_testnet().await?;
```